.github/workflows/release.yml

```diff
@@ -1,10 +1,9 @@
 name: goreleaser
 
 on:
-  pull_request:
   push:
     tags:
-      - '*'
+      - "*"
```

Comment:
What's wrong with single quotes?

Reply:
I blame my auto-formatter. I should look into getting it to be more chill